src/libraries/Microsoft.PowerFx.Core/Functions/Utils.cs

```diff
@@ -16,4 +16,4 @@ public static bool TestBit(this BigInteger value, int bitIndex)
             return !(value & (BigInteger.One << bitIndex)).IsZero;
         }
     }
-}
\ No newline at end of file
+}
```

Comment:
nit: no change to this file?

Reply:
This file has been changed and changed back...
Now, do we need a CR/LF at the end, do I need to change the file, it's hard to find an argument :)